procedures/MT/rotator/rotator_analysis.ipynb

```diff
@@ -0,0 +1,914 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        Analyze the tracking target.
</pre><p>You didn't say what this calculates. Consider:</p><p> </p><p><code>Analyze the tracked target and calculate the RMS of the position error.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done in the commit: [3f0a032](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/3f0a032629e2db3344107ff9821c35efe5e44afd). Thanks!